src/control/server/ctl_storage_rpc.go

```diff
@@ -174,8 +170,10 @@ func (c *ControlService) adjustNvmeSize(resp *ctlpb.ScanNvmeResp) {
 	for _, ctl := range resp.GetCtrlrs() {
 		for _, dev := range ctl.GetSmdDevices() {
 			if dev.GetDevState() != "NORMAL" {
-				c.log.Infof("Skipping unusable device %s (%s): state=%s",
+				c.log.Infof("WARNING: Adjusting available size of unusable SMD "+
```

Comment:
nit - I'd lean toward making this a debug message. I don't know that the "WARNING" portion of the message is really necessary here.

Reply:
- [ ] Update informative message